tests/run/annotation_typing.pyx

```diff
@@ -359,27 +361,46 @@ cdef class ClassTurnOffTyping:
+def return_types_def() -> int:
+    """
+    >>> import inspect
+    >>> inspect.signature(return_types_def).return_annotation
+    'int'
+    """
+
+
+@unittest.skipIf(sys.version_info.major < 3, "Requires Py3+")
+cpdef return_types_cpdef() -> int:
+    """
+    >>> import inspect
+    >>> inspect.signature(return_types_cpdef).return_annotation
+    'int'
+    """
+
+
 _WARNINGS = """
-14:32: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
-14:47: Dicts should no longer be used as type annotations. Use 'cython.int' etc. directly.
-14:56: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
-14:77: Dicts should no longer be used as type annotations. Use 'cython.int' etc. directly.
-14:85: Python type declaration in signature annotation does not refer to a Python type
-14:85: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
-36:40: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
-36:66: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
-63:44: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
-63:70: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
-90:44: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
-90:70: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
-274:44: Unknown type declaration in annotation, ignoring
-302:15: Annotation ignored since class-level attributes must be Python objects. Were you trying to set up an instance attribute?
+16:32: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
+16:47: Dicts should no longer be used as type annotations. Use 'cython.int' etc. directly.
+16:56: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
+16:77: Dicts should no longer be used as type annotations. Use 'cython.int' etc. directly.
+16:85: Python type declaration in signature annotation does not refer to a Python type
+16:85: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
+38:40: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
+38:66: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
+65:44: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
+65:70: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
+92:44: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
+92:70: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
+276:44: Unknown type declaration in annotation, ignoring
+304:15: Annotation ignored since class-level attributes must be Python objects. Were you trying to set up an instance attribute?
```

Comment:
```suggestion
14:32: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
14:47: Dicts should no longer be used as type annotations. Use 'cython.int' etc. directly.
14:56: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
14:77: Dicts should no longer be used as type annotations. Use 'cython.int' etc. directly.
14:85: Python type declaration in signature annotation does not refer to a Python type
14:85: Strings should no longer be used for type declarations. Use 'cython.int' etc. directly.
36:40: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
36:66: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
63:44: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
63:70: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
90:44: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
90:70: PEP-484 recommends 'typing.Optional[...]' for arguments that can be None.
274:44: Unknown type declaration in annotation, ignoring
302:15: Annotation ignored since class-level attributes must be Python objects. Were you trying to set up an instance attribute?
```